Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,53 @@
 import React from 'react';
 import './App.css';
 
+import ActionButton from './components/ButtonComponents/ActionButton';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className = "mainContainer">
+
+        <div className = "div1">
+          <CalculatorDisplay displayStyle = {'display'} text = {'0'} />
+        </div>
+
+      <section className = "textContainer">
+
+        <div className = "div2">
+          <ActionButton actionButton = {'action'} text = {'clear'} />
+          <NumberButton buttonStyle = {'operator'} text = {'/'} />
+        </div>
+
+        <div className = "div3">
+          <NumberButton buttonStyle = {'num'} text = {'7'} />
+          <NumberButton buttonStyle = {'num'} text = {'8'} />
+          <NumberButton buttonStyle = {'num'} text = {'9'} />
+          <NumberButton buttonStyle = {'operator'} text = {'x'} />
```

Comment:
again awesome that you got the objectives done.
a good next step is to look at parts of your code that are being repeated, and find a dynamic way to create components and  stop repeating your code.